code/API_definitions/Edge-Application-Management.yaml

```diff
@@ -925,6 +931,14 @@ components:
         - unknown
       default: unknown
 
+    EdgeCloudZoneFlavors:
+      description: List of unique Name IDs of Infrastructure Flavors
+      type: array
+      items:
+        type: string
+        description: Flavor ID
+        example: A1.2C2M.GPU8G
+
```

Comment:
Shouldn't each flavor have the list and spec of resources they provide?

Reply:
I Agree Nicola, good point. I'll convert the string to object with the spec information. Thanks!